test/pytest/test_utils.py

```diff
@@ -79,16 +79,16 @@ def unregister_model(model_name):
 
 
 def delete_mar_file_from_model_store(model_store=None, model_mar=None):
-    model_store = model_store if (model_store is not None) else f"{ROOT_DIR}/model_store/"
+    model_store = model_store if (model_store is not None) else os.path.join(ROOT_DIR, "model_store")
     if model_mar is not None:
         for f in glob.glob(path.join(model_store, model_mar + "*")):
             os.remove(f)
 
-environment_json = "/../postman/environment.json"
+environment_json = "../postman/environment.json"
```

Comment:
Should we fix this?

Reply:
The stating '/' should be removed as it can be messing with the split occurring in line 91.
Any other change to that line I don't see how it will be necessary.